### DISCUSSION TITLE
update anchor

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1298] [Q&A]
Hi, why I cannot update the new anchor placements ? I press the write to anchor but still in red light.
![VirtualBox_BitcrazeVM 1_20_04_2024_09_43_21](https://github.com/bitcraze/discussions/assets/118436188/1af3b346-c530-4f2d-b65d-4456d271ce23)

**Comment by ToveRumar** [comment]
Hi!
Thanks for your question. 

To figure out what is going wrong,
could you please provide the SW versions you are using for 
CF client, CF lib, positioning node, and CF firmware?
